### PR TITLE
create paroccipital process

### PR DESCRIPTION
**Pull request by megbalk**
issue #2752

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -222010,6 +222010,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500121
+name: paroccipital process
+def: "An occasional process of bone extending downward from the jugular process of the occipital bone." [https://medical-dictionary.thefreedictionary.com/paroccipital+process]
+synonym: "paramastoid process" EXACT []
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222017]: Consider additional synonyms. From https://doi.org/10.1016/j.wneu.2018.06.056:

"The paramastoid process (also termed the paraoccipital, paracondylar, parajugular, and/or estiloid process)[3](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib3), [4](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib4) ... is a bony excrescence, which is an inferior extension of the jugular process of the occipital bone.[3](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib3), [5](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib5)"

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -222010,6 +222010,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500121
+name: paroccipital process
+def: "An occasional process of bone extending downward from the jugular process of the occipital bone." [https://medical-dictionary.thefreedictionary.com/paroccipital+process]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222016]: Re: the use of "occasional":

"It is present in 0.5%−2% of the population..."
From https://doi.org/10.1016/j.wneu.2018.06.056

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -222010,6 +222010,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
...
+synonym: "paramastoid process" EXACT []
+is_a: UBERON:0006983 ! anatomical point
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222018]: I don't think this is an 'anatomical point'.

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comments.

File: src/ontology/uberon-edit.obo (hunk 2 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:222016]: I may change this to a comment and remove the "rare" and "occasional" altogether, as it is only rare in humans.

**Review by bvarner-ebi — APPROVED** [review]

**Comment by megbalk** [comment]
I had to change the db_xref for zygion

**Pull request merged**